lib/kino/data_table.ex

```diff
@@ -49,18 +49,19 @@ defmodule Kino.DataTable do
 
     name = Keyword.get(opts, :name, "Data")
     sorting_enabled = Keyword.get(opts, :sorting_enabled, true)
+    keys = opts[:keys]
 
-    {data_rows, data_columns} =
-      if keys = opts[:keys] do
-        {rows, %{columns: columns}} = Table.to_rows_with_info(tabular, only: keys)
-        nonexistent = keys -- columns
-        {rows, keys -- nonexistent}
+    {data_rows, data_columns, count} =
+      if keys do
+        {rows, info} = Table.to_rows_with_info(tabular, only: keys)
+        nonexistent = keys -- info.columns
+        {rows, keys -- nonexistent, info.count}
       else
-        {rows, %{columns: columns}} = Table.to_rows_with_info(tabular)
-        {rows, columns}
+        {rows, info} = Table.to_rows_with_info(tabular)
```

Comment:
Currently `info` is not exactly the metadata from `Table.Reader.init`, it's a [normalized map](https://github.com/dashbitco/table/blob/950537310ce490d0169c2d5969e00bfd0f02f390/lib/table.ex#L54). We can add `:count` there to be either integer or nil. Alternatively we can return the whole metadata, @josevalim wdyt?

Reply:
Hrm... my preference would be to return the metadata as is. Why do we have to normalize it?

Reply:
I don't think there's a good reason, other than making the info structure more predictable for the end user.